server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -497,24 +498,32 @@ private static void initDirs(VolumeManager fs, UUID uuid, String[] baseDirs, boo
   }
 
   private void initFileSystem(SiteConfiguration siteConfig, Configuration hadoopConf,
-      VolumeManager fs, UUID uuid, String rootTabletDirUri, String rootTabletFileUri)
-      throws IOException {
+      VolumeManager fs, UUID uuid, String rootTabletDirUri, String rootTabletFileUri,
+      String instanceName) throws IOException {
     initDirs(fs, uuid, VolumeConfiguration.getVolumeUris(siteConfig, hadoopConf), false);
 
     // initialize initial system tables config in zookeeper
     initSystemTablesConfig(zoo, Constants.ZROOT + "/" + uuid, hadoopConf);
 
-    VolumeChooserEnvironment chooserEnv = new VolumeChooserEnvironmentImpl(ChooserScope.INIT, null);
+    Text splitPoint = TabletsSection.getRange().getEndKey().getRow();
+
+    VolumeChooserEnvironment chooserEnv = new VolumeChooserEnvironmentImpl(
+        VolumeChooserEnvironment.ChooserScope.INIT, MetadataTable.ID, splitPoint,
+        new ServerContext(siteConfig, instanceName, uuid.toString()));
```

Comment:
The ServerContext object could be re-used.

Reply:
done